### PR TITLE
Move static form html to radio_button_tag

### PR DESCRIPTION
**Pull request by aaron-collier**
Partially Addresses #1024 

This moves the existing raw html (`input type="radio"`) into `radio_button_tag` code in order to set a default checked value on the `Comment Only` option to avoid the `Not Authorized` message.

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: app/views/hyrax/base/_workflow_actions.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
         <% presenter.workflow.actions.each do |key, label| %>
           <div class="radio">
             <label>
-              <input type="radio" name="workflow_action[name]" id="workflow_action_name_<%= key %>" value="<%= key %>">
+              <%= radio_button_tag 'workflow_action[name]', key, :checked => !!key == 'comment_only' %>
```
Comment (jcoyne) [inline app/views/hyrax/base/_workflow_actions.html.erb]: `!!key` is going to cast whatever key is to a boolean, right?  then compare that with the string `'comment_only'` which will always be false? Am I reading that right?

Comment (aaron-collier) [reply app/views/hyrax/base/_workflow_actions.html.erb]: Yup, you're right - DOH - I'll rework that logic.

File: app/views/hyrax/base/_workflow_actions.html.erb [hunk 2]
```diff
@@ -12,7 +12,7 @@
         <% presenter.workflow.actions.each do |key, label| %>
           <div class="radio">
             <label>
-              <input type="radio" name="workflow_action[name]" id="workflow_action_name_<%= key %>" value="<%= key %>">
+              <%= radio_button_tag 'workflow_action[name]', key, :checked => 'checked' %>
```
Comment (jcoyne) [inline app/views/hyrax/base/_workflow_actions.html.erb]: This will make every radio button default to checked.  Shouldn't it only be one of them?

Comment (aaron-collier) [reply app/views/hyrax/base/_workflow_actions.html.erb]: Yeah - i was having trouble with any conditional logic not producing the same result. But I think I've fixed that. The 3rd parameter here needs to be just true/false, not an option for `checked`. It should now only be checked on `comment_only`

**Pull request merged**